### DISCUSSION TITLE
GitHub account sudden suspension

### DISCUSSION THREAD

**Discussion by gensart-x** [community/community#116763] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Long story short. Recently, about a week ago, my account got suspended, I do not know why, I even do not do anything bad (I think). You can check on my GitHub on here : https://github.com/gensart-ai 

![image](https://github.com/community/community/assets/165239315/627ede6c-1aa5-4aac-ac09-9525f2ae3704)

Within with that, I've reported it to GitHub Support for any reason why suspension happened to my account, and until now, no even new update email about my account received to my email. Is the process really like this ? Or should I wait more longer like a month ? or a year ? I understand it might be a while since their Support traffic may be high or something. But, **in a week**, you can do even some minor information update about the account right ?

Beside of that, I'm a bit disappointed with the time of delay, and the sudden suspension without any warning coming to my email, I don't even know why my account got suspended.

Do you guys have any idea what should I do for this ?

**Comment by AidenJamesYt** [comment]
Really all you can do at this time is wait until they review your ticket, which can take up to a few weeks.

**Reply by gensart-x** [reply]
I think so,

**Comment by itspinky95** [comment]
Hi there,

I'm Scott Jones, also known as @itspinky1995 on GitHub. I came across your post today, and it deeply resonated with me as I've encountered the exact same issue. My account was suddenly suspended without any prior warning or explanation, which has left me in a state of confusion and concern.

Like you, I've reached out to GitHub Support in hopes of understanding the reason behind the suspension and seeking a resolution. However, the waiting period without any form of communication or updates has been quite distressing. It's challenging to navigate this uncertainty, especially considering the importance of our GitHub accounts for our work and collaborations.

I noticed you mentioned having contacted GitHub Support about a week ago without receiving any feedback. I'm curious if there have been any developments since then? Have you received any form of response or resolution to your case?

Additionally, I'm at a crossroads about how to proceed. Given your experience, would you recommend creating a new account and moving forward in that direction? I also have another account linked to a previous job, which I'm considering discontinuing.

Any insights or advice you could share based on your journey would be immensely appreciated. It's a tough situation to be in, and hearing from someone navigating the same challenges would be incredibly helpful.

Best,
Scott Jones (@itspinky1995)

**Reply by gensart-x** [reply]
Yeah, from I do ticketing to GitHub and until now, there's no any response for my ticket.

All I can do now is just waiting for the response to know why and how for me to advance.

All my work now temporarily moved to my account now here.

I open the discussion hope to get any more information about why is this happening.

Thanks for your concern to do reply on my discussion.

**Comment by HimanshuW4** [comment]
I also faced this issue on the 26th of March 2024. And since that day, there's been no reply from GitHub. Till that time I have raised the request two times and they've not updated anything apart from the ticket number. I have to create a new account to move forward. There's no option to check the status and the community can't help in this because in simple words it's not their territory to explore.

For the same thing, the GitHub community has mentioned those who have similar issues. Go through this link: https://github.com/orgs/community/discussions/58191

**Reply by gensart-x** [reply]
26 March ?! Same as mine ! I've suspended on that day too. And I have no reply until today too.

I just confused why this thing happened, because there are no even email about the suspension attempt coming to my email.

**Reply by HimanshuW4** [reply]
I am confused too, that why exactly it happened all of sudden. They didn't warn us before nor informed that it's account suspension and not suspicious activity. The team hasn't replied a single thing yet? Due to this my superior thinks that it happens because of third party sites and putting Credentials in there [In short blame indirectly on me for any leak it happens in organisation]. 
- If someone wants to access our account and they failed that doesn't mean that GitHub has right to suspend (we have no part in any of this). Even after having **MFA** (Multi Factor Authentication. It's suspended and we can't do anything about it.
I read it somewhere that it might took 120 days for getting reply from GitHub. But I am not sure of it because it's high volumes and low priority. 
What to do next!!! No update and nothing new happens with any resolution.

**Reply by gensart-x** [reply]
120 days ?! that's awful. I think this type of support should be more improved.

If it still like this, it is not helping.

**Reply by gensart-x** [reply]
I'm beginning to consider other platform for this GitHub.

**Reply by HimanshuW4** [reply]
I am thinking about the Repositories which can be considered as gone, as of now the account is out of our hands. Neither can we log in nor can we get any data and we can't delete our account. Everything revolves around the only thing that reply has to come via GitHub itself. Until then our hands are tight and there's absolutely nothing we can do about it. 120 days, I have read about it somewhere and will try to look for it again. But as far as it's going, it's not clear that will 120 days be enough, or else we stop thinking about it as it was never there.

**Reply by HimanshuW4** [reply]
@gensart-x Check this out: https://github.com/orgs/community/discussions/116566.

**Comment by uranum432** [comment]
If your GitHub account has been suspended, you will need to follow certain steps to resolve the issue and have your account reinstated. Here's what you can do:

Review the suspension email: GitHub usually sends an email explaining the reason for the suspension. Read it carefully to understand why your account was suspended and what actions you need to take to resolve the situation.

Contact GitHub support: Reply to the suspension email or reach out to GitHub support directly through their contact page. Clearly state your situation, provide any relevant information or context, and express your desire to have your account reinstated. Be polite and concise in your communication.

Rectify the issue: If the suspension was due to a specific policy violation or misuse of GitHub's services, you may need to rectify the issue before your account can be reinstated. This might involve removing certain content, addressing violations, or correcting any mistakes you made.

Cooperate with GitHub: If GitHub requires additional information or evidence from you, promptly provide them with the requested details. Be responsive and cooperative during the process, as it can help expedite the resolution.

Wait for a response: GitHub support will review your case and respond to your request. The response time may vary depending on the complexity of the situation and their workload. It's important to be patient during this period.

Follow any additional instructions: If GitHub provides instructions or guidance on specific actions you need to take to reinstate your account, make sure to follow them diligently. This might include agreeing to certain terms, acknowledging mistakes, or taking further actions to resolve the situation.

Maintain a positive approach: Throughout the process, remain professional, cooperative, and respectful in your interactions with GitHub support. A positive attitude can go a long way in resolving the issue and having your account unsuspended.

Remember that these steps are general guidelines, and the specific process to unsuspend your GitHub account may vary depending on the reason for the suspension. It's essential to carefully read the communication from GitHub and follow their instructions accordingly.

**Reply by gensart-x** [reply]
Hi there,

Thanks for the detailed guidance. I've actually followed all the steps you mentioned to resolve the suspension of my GitHub account. I carefully reviewed the suspension email, contacted GitHub support, and cooperated with them by providing all the requested information.

However, it's been a week now, and I haven't received any response or clarification regarding the suspension. I've been diligently checking my email for any updates, but there hasn't been any communication from GitHub, not even a reason for the suspension.Since I've already taken all the necessary steps outlined, I'm now unsure about the status of my account. 

That's why I've opened this discussion, hoping to gain some insight into what might be going on with my account.Your guidance has been helpful, but the lack of communication from GitHub is becoming concerning. 

Do you have any suggestions on what I can do next to resolve this issue?

**Comment by ghostinhershell** [comment]
Hi @gensart-x,

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Reply by gensart-x** [reply]
Hi,

Thanks for the closing reason. If I may have your answer, can you explain this why ?

![Screenshot_2024-04-08-22-40-29-43_e307a3f9df9f380ebaf106e1dc980bb6](https://github.com/community/community/assets/165239315/5c54c7c2-7ac7-46ba-b6d8-615ce7b9fac8)

**Reply by ettaboyle** [reply]
Hi @gensart-x,

The screenshot you included is a notification that is automatically when a user receives a 7-day block from this organization: https://github.com/orgs/community/ 

The GitHub Community Discussions is a space for all GitHub users to come and discuss ideas and projects. This account was flagged and locked because it violated our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. If there is a second offense with that account, it will be blocked indefinitely from this org. 

The screenshot you shared does not include the username that was blocked, so I am unable to share a specific reason. We also cannot share account specific information in this public forum.

**Reply by gensart-x** [reply]
Thanks for the explanation, really.

**Reply by alanzimm85** [reply]
Please help, i need to get my account back, i sent a lot of curriculums with projects on this github account. I didn´t do nothing wrong. It's not from this account, i can´t acces the other. Ticket ID: 2921645

You have been suspended from GitHub.
 
This moderation action was taken after we received a report and determined that your account violates our Acceptable Use Policy prohibiting the use of our platform in direct support of unlawful attacks that cause technical harms. We do not allow anyone to use our platform in direct support of unlawful attacks that cause technical harms, such as using GitHub as a means to deliver malicious executables or as attack infrastructure, for example by organizing denial of service attacks or managing command and control servers. Technical harms means overconsumption of resources, physical damage, downtime, denial of service, or data loss, with no implicit or explicit dual-use purpose prior to the abuse occurring.

When making content moderation decisions, we consider information from a variety of sources, including: account profile data, information contained in submitted reports/notices or discovered through our own voluntarily initiated investigations, and context around the contents of the repository.

If you wish to regain access or would like to dispute that a violation occurred and can provide additional information to show that a different decision should have been reached, please review our [Appeal and Reinstatement Policy](https://docs.github.com/en/site-policy/acceptable-use-policies/github-appeal-and-reinstatement) and submit a request via our [form](https://support.github.com/contact/reinstatement).

**Comment by Naman-Acharya** [comment]
I don't know how but most of users faced similar issue even my account got suspended without any reason. Might be this is because of malware as per the current news but still unaware about it. I hope we get our account back as soon as possible because specially most of my work was on Github and with Github Copilot and all the work was suddenly stopped because of this incident. 

My Ticket No: #2737022

**Comment by alanzimm85** [comment]
Please help, i need to get my account back, i sent a lot of curriculums with projects on this github account. I didn´t do nothing wrong. It's not from this account, i can´t acces the other. Ticket ID: 2921645

You have been suspended from GitHub.
 
This moderation action was taken after we received a report and determined that your account violates our Acceptable Use Policy prohibiting the use of our platform in direct support of unlawful attacks that cause technical harms. We do not allow anyone to use our platform in direct support of unlawful attacks that cause technical harms, such as using GitHub as a means to deliver malicious executables or as attack infrastructure, for example by organizing denial of service attacks or managing command and control servers. Technical harms means overconsumption of resources, physical damage, downtime, denial of service, or data loss, with no implicit or explicit dual-use purpose prior to the abuse occurring.

When making content moderation decisions, we consider information from a variety of sources, including: account profile data, information contained in submitted reports/notices or discovered through our own voluntarily initiated investigations, and context around the contents of the repository.

If you wish to regain access or would like to dispute that a violation occurred and can provide additional information to show that a different decision should have been reached, please review our [Appeal and Reinstatement Policy](https://docs.github.com/en/site-policy/acceptable-use-policies/github-appeal-and-reinstatement) and submit a request via our [form](https://support.github.com/contact/reinstatement).
